webpack/assets/javascripts/react_app/components/hosts/powerStatus/PowerStatusSelectors.js

```diff
@@ -0,0 +1,36 @@
+import { createSelector } from 'reselect';
+import {
+  selectAPIError,
+  selectAPIResponse,
+} from '../../../redux/API/APISelectors';
+
+const selectError = (state, key) => selectAPIError(state, key);
+
+const selectStateFromAPI = (state, key) => selectAPIResponse(state, key).state;
+
+const selectTitleFromAPI = (state, key) => selectAPIResponse(state, key).title;
+
+const selectStatusText = (state, key) =>
+  selectAPIResponse(state, key).statusText;
+
+export const selectState = createSelector(
+  selectStateFromAPI,
+  selectError,
+  (state, error) => (error ? 'na' : state)
+);
+
+export const selectTitle = createSelector(
+  selectTitleFromAPI,
+  selectError,
```

Comment:
`error` is a nested object in the store, reselect works with `===` comparison and in this case, even if the error object is the same (values), it returns false (always )   and no memorization occurs - so this recomputing occurs on every render.

AFAIK we can solve it with two different approaches: 
1. if it will select directly the message (`selectErrorMessage`), which is a string, `===` comparison works by value. 
2.  when merging a nested object to redux (via `merge`), we can use `{deep: true}`, but we need to keep in mind that this is a heavy operation after all.